lbrynet/lbrynet_daemon/auth/server.py

```diff
@@ -114,8 +130,10 @@ def expire_session():
         content = request.content.read()
         try:
             parsed = jsonrpclib.loads(content)
-        except ValueError:
-            return server.failure
+        except ValueError as err:
+            log.error("Unable to decode request json")
```

Comment:
An error here isn't the fault of the daemon so I think this should be a log.warning.